addons/web/static/src/views/view.js

```diff
@@ -208,6 +208,9 @@ export class View extends Component {
             ),
         });
 
+        const technical_chm = useService("technical-chm");
+        useBus(technical_chm.bus, "UPDATE", () => this.render());
```

Comment:
I would remove the bus. As your object is reactive, you can do useState(useService("technical-chm")), and only what should be rendered will be automatically rendered when the value changes